salesagility/SuiteCRM
# Fix #8771 - Silent failure when no PHP-json module installed

## Description
PHP can be compiled without JSON support and this will cause
SuiteCRM to throw 500 errors. Add a check to the installer to
advise the user of the missing dependency.

## Motivation and Context
Bugfix #8771 

## How To Test This
Before PR:
1. Install CentOS 8.1
1. Install PHP etc. (but not the php-json RPM)
1. Observe that the dependencies screen shows no issues
1. Put in valid configuration details
1. Press next, a 500 error is returned to the browser.

After PR:
1. Install CentOS 8.1
1. Install PHP etc. (but not the php-json RPM)
1. The dependencies screen will show a json issue.

## Types of changes

### Final checklist
